src/clojure_lsp/queries.clj

```diff
@@ -36,9 +39,9 @@
   (->> analysis
        (remove-keys #(string/starts-with? (-> % name shared/filename->uri) "file://"))))
 
-(defn ^:private find-first-order-by-project-analysis [pred? analysis]
-  (or (find-first pred? (mapcat val (filter-project-analysis analysis)))
-      (find-first pred? (mapcat val (filter-external-analysis analysis)))))
+(defn ^:private find-last-order-by-project-analysis [pred? analysis]
```

Comment:
this function name is not correct anymore as we find the last but checking the project before, but I can't think in a better name